Gemfile.lock

```diff
@@ -9,65 +9,68 @@ GIT
 GEM
   remote: https://rubygems.org/
   specs:
-    CFPropertyList (3.0.6)
+    CFPropertyList (3.0.7)
+      base64
+      nkf
       rexml
-    activesupport (7.0.7.2)
+    activesupport (5.2.8.1)
```

Comment:
This downgrade has a vulnerability itself, as shown in the [Dependency Review workflow](https://github.com/aws-amplify/amplify-swift/actions/runs/10044254468?pr=3788):

<h2>Vulnerabilities</h2>
<h4><em>Gemfile.lock</em></h4>
<table><tr><th>Name</th><th>Version</th><th>Vulnerability</th><th>Severity</th></tr><tr><td><a href="https://github.com/rails/rails">activesupport</a></td><td>5.2.8.1</td><td><a href="https://github.com/advisories/GHSA-pj73-v5mw-pm9j">Possible XSS Security Vulnerability in SafeBuffer#bytesplice</a></td><td>moderate</td></tr><tr><td colspan="2"><td><a href="https://github.com/advisories/GHSA-j6gc-792m-qgm2">ReDoS based DoS vulnerability in Active Support's underscore</a></td><td>low</td></tr><tr><td colspan="2"><td><a href="https://github.com/advisories/GHSA-cr5q-6q9f-rq6q">Active Support Possibly Discloses Locally Encrypted Files</a></td><td>low</td></tr></table>
<h2>License Issues</h2>